devicemodel/hw/platform/tpm/tpm.c

```diff
@@ -119,7 +119,9 @@ static int init_tpm2_pt(char *opts, struct mmio_dev *tpm2dev)
 	}
 
 	devopts = strdup(opts);
-	vtopts = strstr(devopts,",");
+	if (devopts != NULL) {
+		vtopts = strstr(devopts,",");
+	}
```

Comment:
If devopts is NULL, what should we do? Return directly?

Reply:
If the devopts is NLL, it means there was not enough memory even malloc for a string.
Return is a better way.